src/github.com/kelseyhightower/confd/backends/config-service/client.go

```diff
@@ -57,7 +57,10 @@ func (c *Client) GetValues(keys []string) (map[string]string, error) {
 	for _, v := range keys {
 		bucketsKey := strings.Split(strings.TrimPrefix(v, "/"), "/")
 		buckets := strings.Split(bucketsKey[0], ",")
-		key := bucketsKey[1]
+		var key string
```

Comment:
you don't need this if block. original code is sufficient

Reply:
original code was throwing "runtime error: index out of range" error when bucketsKey has only one element.